src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -719,6 +720,45 @@ private void CheckLiftedUnaryOp(BoundUnaryOperator node)
             }
         }
 
+        private void CheckBinOp(BoundBinaryOperator node)
+        {
+            Debug.Assert(node != null);
+
+            if (node.OperatorKind.IsLifted())
+            {
+                return;
+            }
+
+            if (!_compilation.FeatureStrictEnabled)
```

Comment:
I assume we'll confirm with compat council.

Reply:
I don't think the compat council cares about what happens under the strict flag. It is the purpose of that flag to produce warnings that would otherwise be considered a breaking change. Sort of a poor man's warning wave.

---
In reply to: [205677180](https://github.com/dotnet/roslyn/pull/28869#discussion_r205677180) [](ancestors = 205677180)

Reply:
Ok. I'm fine if we just confirm with @jaredpar (would customers dislike our adding a new warning to the strict flag?).

Reply:
I don't think changes under feature flags should be tracked as breaking changes.